target/linux/imx6/base-files/lib/upgrade/platform.sh

```diff
@@ -64,7 +64,8 @@ platform_check_image() {
 		nand_do_platform_check $board $1
 		return $?;
 		;;
-	xiaomi,dgnwg05lm )
+	xiaomi,dgnwg05lm |\
+	aqara,zhwg11lm )
```

Comment:
Please ensure proper alphabetic sorting (within each "block" and for the blocks).

Reply:
I fixed the sorting and rebased changes on master.